CONTRIBUTING.md

```diff
@@ -0,0 +1,56 @@
+#  Contributing Guideline
+
+### Follow these steps to contribute to the project:
+
+## Step-1 : Forking
+First **Fork** this repository in your github account. 
+
+
+## Step-2 : Cloning
+Now you need to clone your forked repository to your local machine. <br>
+Head back to your **GitHub account**, open the forked repository, click on the *clone* button, click **clone with HTTPS** and then click the *copy to clipboard* icon.
+
+
+Open **Git Bash** (or some other terminal) and type the following git command:
+
+```
+git clone "URL"
+```
+Replace *URL* with the copied link.It should look like the following -
+```
+git clone "https://github.com/Arnab-Shanta-Anu/download-all-repos.git"
+```
+You will see a new folder named *JU-ForkMe* has been downloaded to your machine on the **directory** that *Git Bash* is on.
```

Comment:
plz correct the repo name *JU-ForkMe to download-all-repos

Reply:
Corrected.